src/check-image/check-image.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@nestjs/common';
+import { Message } from 'discord.js';
+
+@Injectable()
+export class CheckImageService {
+  private countsDictionary: { [key: string]: number } = {};
+
+  public checkImage(message: Message): void {
+    const authorId = message.author.id;
+    if (!authorId) {
+      if (this.countsDictionary[authorId]) {
+        this.countsDictionary[authorId]++;
+      } else {
+        this.countsDictionary[authorId] = 1;
+      }
+
+      if (this.countsDictionary[authorId] > 3) {
+        message.author.send('Please consider adding a profile picture.');
```

Comment:
I think a good message might be something like this (we can still keep improving it after it is merged)

```
Please consider adding a profile picture, it is more friendly. It does not have to be you, but something unique, like a cartoon version of yourself or a pet.
```

Reply:
If we can update this text, I think we can merge it